### ISSUE TITLE
final project requirements

### ISSUE THREAD

**Issue by emberian** [emberian/clarkson-cs470#4]
Demo (5 minutes): This part will have 45% of the total project grade. Grading on demo will be for the whole group. The
demo should include the following:
- [x] Run your code for training and show the training accuracy. If the code takes two much time to run, you need to
show the traces of training performed previously. Please be innovative to convince me that your code produces the
training and validation accuracy you claim in the slides.
- [ ] Run the testing with a few samples (at least 5) and show the prediction of your model.
▪ Presentation (8 minutes + 2 min qn. ans.): this step requires you to present some slides. 35% of total project grade is
allotted for this task. Each member of the group should take turn to present 1 or more slides. The grading will be based
on presentation (individual grade) and content in the slides (grade for the group). The presentation should include the
following.
- [ ] Problem overview and examples of practical applications where it can be used
- [x] Dataset description
- [ ] Method details should include what architecture of ML model you used and why. If you have tried multiple model
architectures, please provide description for each.
- [ ] Results of your algorithm on training, validation and test set. Compare the results with the reference (if any).
-[ ] Conclusion should include any insight you gained on the problem, and any suggested future works
